src/main/resources/db/migration/V2024_10_28_1547__CIV-15278_update_case_proceed_offline_defendant_task_item.sql

```diff
@@ -0,0 +1,20 @@
+
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.Update.CaseProceedsInCaseman.Defendant', '{}', '{"Notice.AAA6.CaseProceedsInCaseman.Defendant" : []}'),
```

Comment:
If you want to delete specific notifications (not all), you can add them in notifications_to_delete